server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/RegexMappingRoutes.java

```diff
@@ -59,15 +59,15 @@ public void define(Service service) {
     @Path(BASE_PATH)
     @ApiOperation(value = "adding address-regex mappings to RecipientRewriteTable")
     @ApiResponses(value = {
-        @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "No body created"),
+        @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "Mapping successfully added"),
         @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "Invalid body")
     })
     public HaltException addRegexMapping(Request request, Response response) throws Exception {
         RegexMappingDTO regexMappingDTO = jsonExtractor.parse(request.body());
         MappingSource mappingSource = extractMappingSource(regexMappingDTO);
 
         try {
-            recipientRewriteTable.addRegexMapping(mappingSource, extractRegex(regexMappingDTO));
+            recipientRewriteTable.addRegexMapping(mappingSource, regexMappingDTO.getRegex());
```

Comment:
Why did you remove the `extractRegex` method? I think better to keep it, and to extract the regex above in a variable for more clarity